### DISCUSSION TITLE
Esp32 compatibility

### DISCUSSION THREAD

**Discussion by Arashinof** [RavenSystem/esp-homekit-devices#1511] [Q&A] [answered]
Hi 
is there any way to modify the code to work with esp32? Or do you have any plans for esp32 version?

**Comment by RavenSystem** [comment, marked as answer]
When I have enough free time, I want to support ESP32.

**Reply by Arashinof** [reply]
Thanks :))

**Reply by mwouda** [reply]
Hi RavenSystem, 

Did you have some free time? 
I now have several SONOFF SwitchMan with an ESP32 chip. Can i install esp-homekit firmware?
